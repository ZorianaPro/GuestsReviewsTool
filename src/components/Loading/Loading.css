.Loading p {
  margin-right: 1em;
  display: inline-block;
}

.Loading .Loading-Dot {
  border-radius: 50%;
  width: 0.5rem;
  height: 0.5rem;
  animation: animation-dot-run 1s infinite ease-out;
  background-color: #414141;
  margin-right: 0.25rem;
  display: inline-block;
}

.Loading .Loading-Dot:nth-child(2) {
  animation-delay: 0.22s;
}

.Loading .Loading-Dot:nth-child(3) {
  animation-delay: 0.44s;
}

@keyframes animation-dot-run {
  0%{
    background-color: darkblue;
  }
  50%{
    background-color: cornflowerblue;
  }
  100%{
    background-color: #414141;
  }
}