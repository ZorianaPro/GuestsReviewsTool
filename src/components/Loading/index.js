import React from 'react';
import './Loading.css';

const Loading = () => (
  <span className="Loading">
    <p>Loading</p>
    <span className="Loading-Dot"/>
    <span className="Loading-Dot"/>
    <span className="Loading-Dot"/>
  </span>
);

export default Loading;