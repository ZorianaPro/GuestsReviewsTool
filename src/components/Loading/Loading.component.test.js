import React from 'react';
import { shallow } from 'enzyme';
import Loading from './';

describe('Loading', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <Loading />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<Loading />)).toMatchSnapshot();
  });
});
