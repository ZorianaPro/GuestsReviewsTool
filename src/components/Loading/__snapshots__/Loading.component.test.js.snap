// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders as expected 1`] = `
<span
  className="Loading"
>
  <p>
    Loading
  </p>
  <span
    className="Loading-Dot"
  />
  <span
    className="Loading-Dot"
  />
  <span
    className="Loading-Dot"
  />
</span>
`;
